client/src/pages/ModalBookingScreen.js

```diff
@@ -97,12 +97,16 @@ function ModalBookingScreen({
 			onOpen={() => setOpen(true)}
 			open={open}
 			trigger={
-				<Icon link name="add" size="small" circular inverted color="teal" />
+				// <Icon link name="add" size="small" circular inverted color="teal" />
+				<Icon.Group>
```

Comment:
I think this is missing the pointer change on hover. Maybe it needs to be "link"?